src/lib/stores/current-bible-book.store.ts

```diff
@@ -0,0 +1,3 @@
+import { writable } from 'svelte/store';
+
+export const currentBibleBook = writable<string | null>(null);
```

Comment:
It'd be nice to avoid adding additional global state for this, I'm thinking we could probably use the existing `currentBibleSection` and just derive the currentBibleBook from that in the function below when we need it.